api/src/test/java/org/apache/iceberg/transforms/TestDatesProjection.java

```diff
@@ -77,10 +89,17 @@ public void assertProjectionInclusive(PartitionSpec spec, UnboundPredicate<?> fi
 
     Assert.assertEquals(predicate.op(), expectedOp);
 
-    Literal literal = predicate.literal();
     Dates transform = (Dates) spec.getFieldsBySourceId(1).get(0).transform();
-    String output = transform.toHumanString((int) literal.value());
-    Assert.assertEquals(expectedLiteral, output);
+    if (predicate.op() == Expression.Operation.IN || predicate.op() == Expression.Operation.NOT_IN) {
```

Comment:
Similar to above, inclusive projection never produces NOT_IN, so this should assert that NOT_IN is not produced.